x-pack/plugins/fleet/common/services/package_to_package_policy.test.ts

```diff
@@ -29,7 +28,11 @@ describe('Fleet - packageToPackagePolicy', () => {
         map: [],
       },
     },
-    status: installationStatuses.NotInstalled,
```

Comment:
Not strictly required for this change. Only because I was in here and it's still type safe but w/one less import